src/platform/site-wide/cta-widget/index.js

```diff
@@ -436,12 +455,13 @@ CallToActionWidget.defaultProps = {
 
 const mapStateToProps = state => {
   const profile = selectProfile(state);
-  const { loading, mhvAccount, /* services, */ verified } = profile;
+  const { loading, mhvAccount, /* services, */ verified, status } = profile;
   return {
     // availableServices: new Set(services),
     isLoggedIn: isLoggedIn(state),
     profile: { loading, verified },
     mhvAccount,
+    mviDown: status !== 'OK',
```

Comment:
From what I gathered, the possible statuses for `profile.status` are `OK`, `NOT_FOUND`, `NOT_AUTHORIZED`, and `SERVER_ERROR`.  Are there fringe cases that would cause the status to not be `OK`?

Reply:
Looking at this again, it might be more correct to check that the status is `'SERVER_ERROR'`.